assume/common/base.py

```diff
@@ -807,6 +812,38 @@ def __init__(
         # them into suitable format for recurrent neural networks
         self.num_timeseries_obs_dim = num_timeseries_obs_dim
 
+    def load_actor_params(self, load_path):
```

Comment:
I would rather move the whole LearningStrategy from base to learning_strategies.py instead of adding to much here..

Reply:
@maurerle I believe not all DRL based bidding strategies will be always contained within a single file. For now we already have learning_strategies.py and learning_advanced_startegies.py. So your suggestions would not work. But overall it is a good idea to maybe move it somewhere else from the base, but where...